extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -95,6 +95,12 @@ export const requiredIpykernelPkg: PythonPkgDetails = {
 	installExactVersion: true
 };
 
+export const requiredTraitletsPkg: PythonPkgDetails = {
```

Comment:
Actually, while you're here could you link the issue explaining this (and then do the same for the notebook package too)?